packages/terra-clinical-result/src/terra-dev-site/doc/example/flowsheetresultcell/FlowsheetResultCellBloodPressure.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { FlowsheetResultCell } from 'terra-clinical-result/lib/index';
+import classNames from 'classnames/bind';
+import styles from '../Examples.module.scss';
+
+const cx = classNames.bind(styles);
+
+const bloodpressureResultValue = [
+  {
+    id: '111',
+    systolic: {
+      eventId: '111.1',
+      result: {
+        value: '111',
+        unit: 'mmHg',
+      },
+      interpretation: 'CRITICAL',
+      resultDateTime: '10/10/2020 11:11',
```

Comment:
What's the difference between `resultDateTime` and `datetimeDisplay`? I see some systolic results use `resultDateTime` while other systolic results use `datetimeDisplay`.